pkg/server/server.go

```diff
@@ -456,6 +456,10 @@ func (s *Server) handleHelmList(user *auth.User, w http.ResponseWriter, r *http.
 		sendResponse(w, http.StatusBadGateway, apiError{fmt.Sprintf("Failed to list helm releases: %v", err)})
 	}
 	w.Header().Set("Content-Type", "application/json")
-	res, _ := json.Marshal(resp)
-	w.Write(res)
+	if resp == nil {
```

Comment:
we have changed the contract of `ListRelease` above to return empty list, so this condition is now impossible to get?

Reply:
yes removed this